images/installer/Dockerfile.rhel7

```diff
@@ -0,0 +1,50 @@
+# It builds an image containing only the openshift-install and terraform binaries.
+
+FROM rhel7:latest as builder
+
+USER root
+ENV GOPATH="/go"
+ENV PATH="$PATH:$GOPATH/bin:/usr/local/go/bin"
+ENV INSTALLERDIR="$GOPATH/src/github.com/openshift/installer"
+ENV GOVERSION="1.10.5"
+ENV DEPVERSION="0.5.0"
+
+# Packages and binaries requisites
+# golang 3.10 is not available in RHEL7
+RUN yum install gcc-c++ git -y && \
+    mkdir -p ~/{tmp,bin} && \
+    curl -L https://dl.google.com/go/go${GOVERSION}.linux-amd64.tar.gz -o ~/tmp/go.tar.gz && \
+    tar -C /usr/local -xzf ~/tmp/go.tar.gz && \
+    rm -f ~/tmp/go.tar.gz && \
+    curl -L https://github.com/golang/dep/releases/download/v${DEPVERSION}/dep-linux-amd64 -o /usr/local/bin/dep && \
+    chmod a+x /usr/local/bin/dep
+
+# Clone the repo
+RUN mkdir -p $GOPATH/{src,bin,pkg} && \
+    mkdir -p $GOPATH/src/github.com/openshift/ && \
+    git clone https://github.com/openshift/installer.git $INSTALLERDIR
+
+# Dependencies installation
+RUN cd $INSTALLERDIR && \
+    dep ensure -v
```

Comment:
If all we're doing is building the installer, we shouldn't need `dep`.  `dep` is a tool for keeping `vendor/` up to date, so it matters for folks pushing new installer commits.  But it is not needed for folks building the installer from an existing commit.

Reply:
Thanks for the suggestion, I've pushed a new version without dep and with golang installed from the SCL.